src/main/java/com/mongodb/hibernate/internal/extension/service/StandardServiceRegistryScopedState.java

```diff
@@ -33,13 +33,17 @@
 import java.util.Map;
 import java.util.Set;
 import org.hibernate.HibernateException;
+import org.hibernate.boot.registry.BootstrapServiceRegistry;
 import org.hibernate.boot.registry.StandardServiceInitiator;
+import org.hibernate.boot.registry.StandardServiceRegistry;
 import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
 import org.hibernate.service.Service;
 import org.hibernate.service.UnknownServiceException;
 import org.hibernate.service.spi.ServiceRegistryImplementor;
 import org.jspecify.annotations.Nullable;
 
+/** @mongoCme Thread-safe. */
```

Comment:
In most other places I said "Must be thread-safe", but here it is just "Thread-safe". The difference is that here the behavior is something we provide based on what we need. In other places we must provide thread-safety because of the Hibernate ORM API parts we implement/extend.